openaerostruct/aerodynamics/aero_groups.py

```diff
@@ -30,6 +30,37 @@ def setup(self):
         surfaces = self.options["surfaces"]
         rotational = self.options["rotational"]
 
+        # Check for multi-section surfaces and create suitable surface dictionaries for them
+        for i, surface in enumerate(surfaces):
+            # If multisection mesh then build a single surface with the unified mesh data
+            if "isMultiSection" in surface.keys():
+                import copy
+
+                target_keys = [
+                    # Essential Info
+                    "name",
+                    "symmetry",
+                    "S_ref_type",
+                    "ref_axis_pos",
+                    "mesh",
+                    # aerodynamics
+                    "CL0",
+                    "CD0",
+                    "with_viscous",
+                    "with_wave",
+                    "groundplane",
+                    "k_lam",
+                    "t_over_c_cp",
+                    "c_max_t",
+                ]
+
+                # Constructs a surface dictionary and adds the specified supported keys and values from the mult-section surface dictionary.

```

Comment:
It doesn't eliminate them from the original `surface` dict. The issue is that the the multisurface dict is constructed a bit differently than the single surface dict. Giving the multisurface dict to AeroPoint caused issues so what I do is rather than modify AeroPoint, I give AeroPoint a version of the multisurface dict that only contains information necessary for AeroPoint(basically none of the geometry parameterization stuff). The purpose of this approach is to avoid having to make significant changes to the core components of OAS. I don't see anyone needing the geometry parameterization data downstream of AeroPoint. If it's needed for what ever reason they can get it from the Geometry Group itself.